libraries/core_libs/consensus/include/transaction/transaction_queue.hpp

```diff
@@ -122,6 +133,13 @@ class TransactionQueue {
 
   ExpirationCache<trx_hash_t> known_txs_;
 
+  // Last time transactions were dropped due to queue reaching max size
+  uint64_t transaction_overflow_time_ = 0;
```

Comment:
use std::chrono::system_clock::time_point so it is clear also by looking at type that it is time point. Then also this function get much cleaner:

```
bool transactionsDropped() const {
    return (std::chrono::duration_cast<std::chrono::seconds>(std::chrono::system_clock::now().time_since_epoch())
                .count() -
            transaction_overflow_time_) < kTransactionOverflowTimeLimit;
  }
```

as you can simply use now() - transaction_overflow_time_